### PR TITLE
[test] Remove PrgEnv-pgi from FFTW test

### PR DESCRIPTION
**Pull request by jjotero**
FFTW checks are no longer building with PrgEnv-pgi on daint after the the pgi/20.x.x was made default.
Cray does not support FFTW with the PGI compiler, so we remove PrgEnv-pgi from the valid environments for this test on daint.

### CODE REVIEW

**Comment by victorusu** [comment]
@jenkins-cscs retry all

**Review by victorusu — APPROVED** [review]
`lgtm`

File: cscs-checks/microbenchmarks/fft/fftw_benchmark.py [hunk 1]
```diff
@@ -24,10 +24,10 @@ def __init__(self, exec_mode):
             self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-pgi']
             self.build_system.cflags += ['-I$FFTW_INC', '-L$FFTW_DIR',
                                          '-lfftw3']
-        elif self.current_system.name in {'daint', 'tiger'}:
+        elif self.current_system.name in {'tiger'}:
```
Comment (vkarak) [inline cscs-checks/microbenchmarks/fft/fftw_benchmark.py:27]: You can put Tiger in the same list as Daint and Dom.

File: cscs-checks/microbenchmarks/fft/fftw_benchmark.py [hunk 2]
```diff
@@ -24,10 +24,10 @@ def __init__(self, exec_mode):
...
             self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-pgi',
                                         'PrgEnv-gnu']
-        elif self.current_system.name == 'dom':
+        elif self.current_system.name in {'daint', 'dom'}:
             # Cray FFTW library is not officially supported for the PGI
             # compiler (Cray case #266199)
```
Comment (vkarak) [inline cscs-checks/microbenchmarks/fft/fftw_benchmark.py:32]: Can you remove also the Cray case number? 

```suggestion
            # Cray FFTW library is not supported for the PGI compiler
```

**Review by vkarak — CHANGES_REQUESTED** [review]

**Review by vkarak — APPROVED** [review]
lgtm

**Pull request merged**